src/Controllers/SCStormRunoffController.ts

```diff
@@ -1548,13 +1630,15 @@ module StreamStats.Controllers {
         }
 
         private formatCSV() {
-            //ga event
-            this.angulartics.eventTrack('Download', { category: 'Report', label: 'CSV' });
-
             var filename = 'data.csv';
 
             var BohmanRural1989 = () => {
-
+                var finalVal = 'USGS SC Flood Hydrograph for Rural Watersheds using ' + this.SelectedAEP.name + ' AEP\n';
+                finalVal += '\n' + "Warning Messages:," + this.warningMessages + '\n';
+                finalVal += this.tableToCSV($('#BohmanRuralParameterTable'));
+                finalVal += '\n' + this.tableToCSV($('#BohmanRuralSummaryTable'));
+                finalVal += '\n\n' + this.tableToCSV($('#BohmanRuralHydrograph'));
```

Comment:
```suggestion
                finalVal += '\n\nTabular Hydrograph';
                finalVal += '\n\n' + this.tableToCSV($('#BohmanRuralHydrograph'));
```

Reply:
Adds a title to the hydrograph table. Do you mind making this change for Bohman Urban as well?